.github/scripts/generate_binary_build_matrix.py

```diff
@@ -106,7 +106,7 @@
     "13.0": (
         "cuda-bindings==13.0.3; platform_system == 'Linux' | "
         "nvidia-cuda-nvrtc==13.0.88; platform_system == 'Linux' | "
-        "nvidia-cuda-runtime==13.0.96; platform_system == 'Linux' | "
+        "nvidia-cuda-runtime~=13.0.96; platform_system == 'Linux' | "
```

Comment:
If I understand [PEP-0440](https://peps.python.org/pep-0440/#compatible-release) correctly, we want something like this
```suggestion
        "nvidia-cuda-runtime==13.0.*; platform_system == 'Linux' | "
```